src/render/src/pso/buffer.rs

```diff
@@ -104,19 +104,29 @@ pub struct ConstantBuffer<T: Structure<shade::ConstFormat>>(
 #[derive(Clone, Debug, Eq, Hash, PartialEq)]
 pub struct RawConstantBuffer(Option<(Usage, ConstantBufferSlot)>);
 
-/// Global (uniform) constant component. Describes a free-standing value passed into
-/// the shader, which is not enclosed into any constant buffer. Deprecated in DX10 and higher.
+/// Global (uniform) constant component. Describes a free-standing value passed
+/// into the shader, which is not enclosed into any constant buffer. Deprecated
+/// in DX10 and higher.
 ///
 /// - init: `&str` = name of the constant
 /// - data: `T` = value
 #[derive(Derivative)]
 #[derivative(Clone, Debug, Eq, Hash, PartialEq)]
 pub struct Global<T: ToUniform>(
-    Option<shade::Location>,
+    RawGlobal,
     #[derivative(Hash = "ignore", PartialEq = "ignore")]
     PhantomData<T>
 );
 
+/// Raw global (uniform) constant component. Describes a free-standing value
```

Comment:
let's explain a little on the difference between this and a regular `Global`.
Also, let's remove the "Deprecated in ..." comment both here and above, since you are at it ;) thanks!